### PR TITLE
vweb.csrf: adding README.md, corrected comments

### PR DESCRIPTION
**Pull request by floscodes**
I added README.md to the vweb.csrf-module. I also corrected some comments in the code due to spelling mistakes.

### CODE REVIEW

File: vlib/vweb/csrf/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# vweb.csrf - Provides protection against Cross-site request forgery (CSRF) for web apps written with vweb
+
+## Usage
+
+When building a csrf-protected service, first of all create a `struct`that implements `csrf.App`
+
+```
```
Comment (enghitalo) [inline vlib/vweb/csrf/README.md:7]:
```v ignore

File: vlib/vweb/csrf/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+// embeds the csrf.App struct in order to empower the struct to protect against CSRF
+struct App {
+	csrf.App
+}
+```
+
+Start a server e.g. in the main function.
+
+```
```
Comment (enghitalo) [inline vlib/vweb/csrf/README.md:21]:
```v ignore

File: vlib/vweb/csrf/README.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        port: 8080
+    }) or { panic(err) }
+}
+```
+
+### Enable CSRF-protection
+
+Then add a handler-function to define on which route or on which site the CSRF-Token shall be set.
+
+```
```
Comment (enghitalo) [inline vlib/vweb/csrf/README.md:33]:
```v ignore

**Review by enghitalo — CHANGES_REQUESTED** [review]

File: vlib/vweb/csrf/README.md (hunk 2 above)
Comment (enghitalo) [reply vlib/vweb/csrf/README.md:21]:
```v ignore

File: vlib/vweb/csrf/README.md (hunk 1 above)
Comment (enghitalo) [reply vlib/vweb/csrf/README.md:7]: v ignore

File: vlib/vweb/csrf/README.md (hunk 2 above)
Comment (enghitalo) [reply vlib/vweb/csrf/README.md:21]: v ignore

**Comment by enghitalo** [comment]
@floscodes you can see and accept
https://github.com/floscodes/v/pull/1

**Comment by floscodes** [comment]
Ok, so is it more recommended to solve this by using "v ignore"? Now I solved it by making some new lines in the md-File and it worked... shall I change it an merge your version, @enghitalo ?

**Comment by enghitalo** [comment]
> Ok, so is it more recommended to solve this by using "v ignore"? Now I solved it by making some new lines in the md-File and it worked... shall I change it an merge your version, @enghitalo ?
> 
The v ignore is used to set the v highlights in the github markdown files

**Comment by floscodes** [comment]
Ok I see! Thanks!

File: vlib/vweb/csrf/README.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+	token := app.get_csrf_token() or { panic(err) }
+
+	return app.text("Csrf-Token set! It's value is: $token")
+}
+```
+
+If you want to set the cookies's HttpOnly-status to false in order to make it  
+ accessible to scripts on your site, you can do it like this:
+`app.set_csrf_cookie(csrf.HttpOnly{false})`
+If no argument is passed the value will be setted to true by default.
```
Comment (spytheman) [inline vlib/vweb/csrf/README.md:50]:
```suggestion
If no argument is passed the value will be set to true by default.
```

**Pull request merged**